.travis.php

```diff
@@ -21,6 +21,10 @@
 
     $json = file_get_contents($dir.'/composer.json');
     $package = json_decode($json);
+    if (!$package || '{' !== $json[0]) {
+        echo "Invalid composer.json\n";
```

Comment:
No need, we must get an object here, and error reporting is helpless without a custom json parser